packages/app/src/state/cortex-config/sagas/cortexConfigSubscription.ts

```diff
@@ -116,7 +116,7 @@ export function cortexConfigSubscriptionEventChannel(): EventChannel<Actions> {
             emitter(actions.setCortexRuntimeConfig(res.data));
           })
           .catch(err => {
-            if (ServerError.isInstance(err.response.data)) {
+            if (axios.isAxiosError(err) && err.response && ServerError.isInstance(err.response.data)) {
```

Comment:
`ServerError.isInstance(err.response.data)` alone will break if an generic, non-request error is thrown, as it doesn't have the `response` field.